src/components/measuretool/MeasureResult.vue

```diff
@@ -1,51 +1,59 @@
 <template>
 
   <div class="">
-    <div class="measure-result">
+    <div class="measure-result" v-if="measureType === 'distance' || measureType === 'area'">
       {{ $t("wgu-measuretool.length") }}: {{distance}}
     </div>
-    <div class="measure-result">
+    <div class="measure-result" v-if="measureType === 'area'">
       {{ $t("wgu-measuretool.area") }}: {{area}}
     </div>
+    <div class="measure-result" v-if="measureType === 'angle'">
+      {{ $t("wgu-measuretool.angle") }}: {{angle}}
+    </div>
   </div>
 
 </template>
 
 <script>
+import AngleUtil from '../../../src/util/Angle';
 import LineStringGeom from 'ol/geom/LineString';
-import PolygonGeom from 'ol/geom/Polygon';
 import { getArea, getLength } from 'ol/sphere.js';
 
 export default {
   name: 'wgu-measure-result',
   props: {
-    measureGeom: { type: Object }
+    measureGeom: { type: Object },
+    measureType: { type: String }
   },
   data () {
     return {
       area: ' -- ',
-      distance: ' -- '
+      distance: ' -- ',
```

Comment:
the string ` -- ` appears quite often and could be extracted to an constant. But that's more a matter of taste ...